### DISCUSSION TITLE
Access user-specific song stats (like Recap data)?

### DISCUSSION THREAD

**Discussion by deadhead-pharaoh** [sigma67/ytmusicapi#368] [Q&A] [answered]
Any insight into how to access user-specific song metadata like lastPlayedDate or playCount (made up names but you get the idea). This user-specific data must be tracked somewhere as YTM creates the "Recap" stuff from it, I assume. Thoughts?

**Comment by sigma67** [comment, marked as answer]
If you can find it somewhere in the web interface it can be accessed, else there's probably not a way

**Comment by deadhead-pharaoh** [comment]
Thanks, just figured I'd follow-up: The info I'm after doesn't seem to be surfaced in the web interface so this doesn't seem possible to do yet, but with the changes YTM has been making lately I'm hoping maybe one day we'll be able to access them.  I have a huge (>10k) Thumbs-Up list & would love to be able to make some "smart" variants of it (i.e. a sub-playlist only of songs I've not heard in the last 30 days, etc., that kinda thing).
